Cargo.toml

```diff
@@ -18,10 +18,11 @@ v1_32_4 = ["v1_32", "pango-sys/v1_32_4"]
 v1_36_7 = ["v1_34", "pango-sys/v1_36_7"]
 v1_34 = ["v1_32_4", "pango-sys/v1_34"]
 v1_31 = ["pango-sys/v1_31"]
+dox = []
```

Comment:
Is this how it is supposed to be? gtk and gio look like this

Reply:
Not exactly. It needs to trigger the dox feature of `pango-sys` (and of `glib`, I think?).

Reply:
Better `dox = ["pango-sys/dox", "glib/dox"]` as in https://github.com/gtk-rs/gio/pull/57

Reply:
So just like I said. :D Thanks for the confirmation @EPashkin!

Reply:
Why also of glib? For the -sys crate it makes sense because it uses feature-gated API there

Reply:
And then various crates needs to be fixed, before my regens already :)

Reply:
Yep, sorry. Was doing it veeeeeeeery slowly.

Reply:
@GuillaumeGomez Only needed `cairo` crate, not sure what it need be in `dox` as `glib` is optional.

Reply:
So should each crate's dox feature only depend on the -sys/dox feature?

Reply:
IMHO gdk is exception it need include gio, pango, cairo and maybe glib too.

Reply:
Ok, why? :) Someone please fix it for me or explain to me how it works ;)

Reply:
My opinion: if someone try to build docs only for gdk he get all functions for it but only part for dependencies.
Also as gtk will depends only for gdk it help also will be partial.

Reply:
It's in order to overpass the rust's limitations so we can get docs from every functions and not only the ones from the archs/version.

Reply:
Yes but for that it's enough to do have $crate's dox feature to depend on $crate-sys/dox? Or should every possible dependency be added there?

Reply:
Well, if you don't pass the parameters to your non-sys dependencies, then it won't have the correct rendering (the non-dox one).

Reply:
Ok, but that's not very ergonomic then. You would need to list each of your dependencies again then for completeness :(

What do you want to have added here?

Reply:
Should be enough, yes.

Reply:
Ok, perfect :)